test/unit/renderer/viz/expressions/ramp.test.js

```diff
@@ -975,7 +975,7 @@ describe('src/renderer/viz/expressions/ramp', () => {
                 const r = ramp(zoomrange([3, 9]), [100, 200]);
                 r._bindMetadata();
                 const fakeGL = {uniform1f: () => {}, uniform1i: () => {}};
-                const fakeDrawMetadata = {zoom: 1000.0};
+                const fakeDrawMetadata = {zoomLevel: 100};
```

Comment:
Why this change in magnitude?

Reply:
With `1000.0` the test pass too. I just changed it for aesthetics and because since now the zoomLevel is in other range (logarithmic) using such a big value seemed to me like a sign of not using the logarithmic level.

Reply:
Oh I see, thx for the explanation 👍